client/src/pages/signup/SignupForm.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  TextField,
+  Paper,
+  Typography,
+  makeStyles,
+  Button,
+  Link
+} from "@material-ui/core";
+import { Link as RouterLink } from "react-router-dom";
+
+//CSS styles
+const useStyles = makeStyles({
+  container: {
+    width: "490px",
+    height: "500px",
+    margin: "40px auto 0 auto",
+    padding: "2rem 4rem 2rem 4rem",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  title: {
+    marginBottom: "20px"
+  },
+  buttonContainer: {
+    flexGrow: 1
+  },
+  button: {
+    width: "10rem",
+    margin: "1rem"
+  },
+  textField: {
+    marginTop: "10px"
+  }
+});
+
+const SignupForm = ({ fields, signup }) => {
```

Comment:
SignupForm and LoginForm look very similar... maybe you want to combine them into one component that is the paper that can go into the common folder

Reply:
Updated. Combined LoginForm and SignupForm to one component